cmd/buildah/containers.go

```diff
@@ -90,6 +90,9 @@ var (
 )
 
 func containersCmd(c *cli.Context) error {
+	if len(c.Args()) > 0 {
+		return errors.New("'buildah containers' accepts no arguments")
```

Comment:
I think I'd prefer "does not accept arguments" over "accepts no arguments" for both of these errors.

Reply:
I'm sitting on the fence about the need for this change or just changing it to a warning message and then just continue on.  @rhatdan WDYT?

Reply:
I prefer Tom's message.  I am fine with this error-ing out.

Reply:
updated, PTAL.